.travis.yml

```diff
@@ -18,6 +18,8 @@ matrix:
       env: SYMFONY_VERSION=2.4.*
     - php: 5.5
       env: SYMFONY_VERSION=2.6.*
+    - php: 5.5
+      env: SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
we can't change the xml configurations while keeping 2.3 support. That's why we still have deprecation errors. As these can't be fixed, I've set the mode to weak to not make the tests fail because of it.